### ISSUE TITLE
Nested heading followed by content should not fail "content must follow headings"

### ISSUE THREAD

**Issue by joshski** [bbc/bbc-a11y#106]
This scenario fails:
```
  Scenario: Nested heading followed by nested content
    Given a page with the body:
      """
      <div>
        <h1>Main heading</h1>
      </div>
      Followed by some content
      """
    When I validate the "content must follow headings" standard
    Then it passes
```
I think it should pass. Can somebody please confirm.

**Comment by EmmaJP** [comment]
Yes, that should probably pass ... though it may be worth determining that "Followed by some content" is not another <h1>.

**Comment by joshski** [comment]
Good point, I'll add that too.

**Comment by joshski** [comment]
Fixed in c5c4b1a496cde976298125f1e9a82124326cac18
